strr-examiner-web/app/stores/examiner.ts

```diff
@@ -5,6 +5,8 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
 
   const tableLimit = ref(50)
   const tablePage = ref(1)
+  const isApplication = ref<boolean>(true)
```

Comment:
I would make it computed based on activeRecord, so we don't have to worry about setting it in existing and future fetch calls.

Reply:
Done